web/db.py

```diff
@@ -656,7 +656,7 @@ def __init__(self, db_module, keywords):
         self.supports_multiple_insert = False
 
         try:
-            import DBUtils  # noqa, flake8 F401
+            import dbutils  # noqa: F401
 
             # enable pooling if DBUtils module is available.
```

Comment:
Let’s just simplify…
```python
def get_pooled_db():
            from dbutils import pooled_db as PooledDB
            return PooledDB.PooledDB(creator=self.db_module, **keywords)
```

Reply:
Also, given that the `ast` module has be around since [Python 2.6](https://docs.python.org/2.6/library/ast.html), can you please delete lines 21-25 and then put `import ast` near the top of the file?

Reply:
I don't quite understand regarding your first suggestion. Do you want to remove optional pooling? So, `dbutils` is always necessary? Or, do you just want to get the functionality into a separate function? I suspect it's the later, but just want to confirm.